docs/fuzzer.md

```diff
@@ -0,0 +1,21 @@
+# Fuzzing
+
+We are using the [LLVM Fuzzer](https://llvm.org/docs/LibFuzzer.html).
+
+The fuzzer used here is without structured input generation. Instead, we rely 
+on the fuzzer to mutate the input.
+
+The initial seeds are all the ELF files in the build folder.
+
+## Execution
+
+- `./.ci/riscv-fuzz.sh`
```

Comment:
Use shorter form. e.g., `.ci/fuzz.sh`

Reply:
Updated!

I was following the convention in the `.ci` folder, thus added the `riscv-` prefix.